JULIERAJ/KIDS-FIRST-Front
# Fix/24/social buttons checkbox

## Sign up/in Styling

## Requirements
- Google/facebook buttons and checkbox should follow the design in Figma

## Notes
- Issue 20 has been also resolved in this ticket
- Refactoring: unneeded divs have been removed
- Logo and forms layout have been fixed

## Test Steps
- Compare with the Figma design to check the styling matches in Login and Register pages

## UI/UX Outcome
<img width="1711" alt="Screenshot 2024-07-12 at 13 00 15" src="https://github.com/user-attachments/assets/245193ba-ffcf-45bc-b7fc-88f84f9fb313">
<img width="1706" alt="Screenshot 2024-07-12 at 13 00 30" src="https://github.com/user-attachments/assets/2c2e9596-eb00-4c42-a6cf-9f053a03889f">
<img width="1709" alt="Screenshot 2024-07-12 at 13 01 07" src="https://github.com/user-attachments/assets/bc8ae847-32f4-4b55-bdac-7fdff68c9ed4">

## Checklist
